js/game.js

```diff
@@ -506,6 +507,18 @@ function displayLeaderboard() {
     ]
 
 
+    const tbody = document.getElementById('leader-table-body');
+    leaderData.forEach((leader)=>{
+        const row = document.createElement('tr');
+        tbody.appendChild(row);
+        const userData = document.createElement('td');
+        const avgData = document.createElement('td');
```

Comment:
Why did you use this variable name?

Reply:
I was just following directions, I promise

Reply:
ok sounds good to me